src/index.js

```diff
@@ -172,43 +176,35 @@ const monitorTizenPlayer = function (player, options) {
       } else if (eventType === 'PLAYER_MSG_HTTP_ERROR_CODE') {
         // Note: This event has the same problem as PLAYER_MSG_FRAGMENT_INFO.
       }
-      if (player.playbackCallback && player.playbackCallback.onevent) {
-        setTimeout(() => {
-          player.playbackCallback.onevent(eventType, eventData);
-        }, 0);
-      }
+      setTimeout(() => {
+        player.playbackListener.onevent(eventType, eventData);
+      }, 0);
     },
 
     onerror: function (eventType) {
       if (!options.automaticErrorTracking) { return; }
       player.mux.emit('error', { player_error_code: -1, player_error_message: eventType });
-      if (player.playbackCallback && player.playbackCallback.onerror) {
-        setTimeout(() => {
-          player.playbackCallback.onerror(eventType);
-        }, 0);
-      }
+      setTimeout(() => {
+        player.playbackListener.onerror(eventType);
+      }, 0);
     },
 
     ondrmevent: function (drmEvent, drmData) {
-      if (player.playbackCallback && player.playbackCallback.ondrmevent) {
-        setTimeout(() => {
-          player.playbackCallback.ondrmevent(drmEvent, drmData);
-        }, 0);
-      }
+      setTimeout(() => {
+        player.playbackListener.ondrmevent(drmEvent, drmData);
+      }, 0);
     },
 
     onsubtitlechange: function (duration, text, type, attriCount, attributes) {
-      if (player.playbackCallback && player.playbackCallback.onsubtitlechange) {
-        setTimeout(() => {
-          player.playbackCallback.onsubtitlechange(duration, text, type, attriCount, attributes);
-        }, 0);
-      }
+      setTimeout(() => {
+        player.playbackListener.onsubtitlechange(duration, text, type, attriCount, attributes);
+      }, 0);
```

Comment:
Curious why you're running these callbacks on the next tick of the event loop, I wouldn't think that is needed